Docs/spack.yaml

```diff
@@ -0,0 +1,26 @@
+# This is a Spack environment file.
+#
+# This environment can be used to install all dependencies to build the manual
+# locally.
+#
+# Activating and installing this environment will provide all dependencies
+# that are needed for full-feature development.
+#   https//spack.readthedocs.io/en/latest/environments.html#anonymous-environments
```

Comment:
Is this link supposed to point to a page in `homeimprovement.com`?
My browser (`safari`) points to that when I use this link in it.

Reply:
Oh no. :-o I forgot a `:` and your internet provider seems to capture unused domains to show you ads.
```suggestion
#   https://spack.readthedocs.io/en/latest/environments.html#anonymous-environments
```

Reply:
😆

Reply:
Thanks for catching that one :sweat_smile: